qiita_pet/handlers/study_handlers/listing_handlers.py

```diff
@@ -35,41 +39,78 @@ def _get_shared_links_for_study(study):
     return ", ".join(shared)
 
 
-def _build_study_info(studytype, user=None):
+def _build_study_info(studytype, user, results=None):
     """builds list of namedtuples for study listings"""
-    if studytype == "private":
-        studylist = user.user_studies
-    elif studytype == "shared":
-        studylist = user.shared_studies
-    elif studytype == "public":
-        studylist = Study.get_by_status('public')
-    else:
+    if studytype not in {"standard", "shared"}:
         raise IncompetentQiitaDeveloperError("Must use private, shared, "
                                              "or public!")
-
-    StudyTuple = namedtuple('StudyInfo', 'id title meta_complete '
-                            'num_samples_collected shared num_raw_data pi '
-                            'pmids owner status abstract')
+    # get list of studies for table
+    if studytype == "standard":
+        studylist = user.user_studies | Study.get_by_status('public')
```

Comment:
if possible, it would be nice to stick with either operators (this line), or methods (line 53), to improve consistency within the method being worked on.